content/en/docs/reference/command-line-tools-reference/feature-gates/watch-list-client.md

```diff
@@ -0,0 +1,18 @@
+---
+title: WatchListClient
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.30"
+---
+This enables an API client (inlcuding some control plane components) to
+request a stream of data rather than an entire list. The behavior change
+is implemented in the client-go library and it is opaque to the client.
```

Comment:
Typo in “inlcuding“.
```suggestion
Enable selected control plane components to fetch collections using **watch** rather
than **list** requests, as an optimization. You must ensure that the authorization rules
and checks that your cluster makes will allow these requests. Kubernetes' default rules,
including for the [RBAC](/docs/reference/access-authn-authz/rbac/) authorization mode,
_do_ allow using **watch** in place of **list**.

```


(aside: once we default this to true, that is then an important change to mention in the release announcement article, as it could break some clusters)

Reply:
Em ... why we want to touch the authorization topic in this feature gate?

Reply:
I checked and actually the control plane should be able to fall back to **list**, so no authz changes are needed.